### PR TITLE
[ci] profiles: last-rite net-misc/netkit-rsh

### PR DESCRIPTION
**Pull request by ajakk**
Signed-off-by: John Helmert III <ajak@gentoo.org>

### CODE REVIEW

File: profiles/base/package.use.force [hunk 1]
```diff
@@ -206,3 +206,8 @@ dev-lang/python wide-unicode
 # Forcing w.r.t. bug 265336. When unicode use-flag is
 # turned off, ABI is broken without a .so bump.
 dev-libs/libpcre unicode
+
+# John Helmert III <ajak@gentoo.org) (2021-06-19)
+# netkit-rsh is last-rited and torque has
+# '!crypt? ( net-misc/netkit-rsh )', so force USE=crypt
+net-misc/netkit-rsh crypt
```
Comment (SoapGentoo) [inline profiles/base/package.use.force:213]: missing final EOL

**Review by SoapGentoo — APPROVED** [review]

File: profiles/base/package.use.force (hunk 1 above)
Comment (ajakk) [reply profiles/base/package.use.force:213]: Well, that's supposed to force crypt on torque, so it's a typo anyway

Comment (thesamesam) [reply profiles/base/package.use.force:213]: (Fix email too)

Comment (ionenwks) [reply profiles/base/package.use.force]: Rather than EOL, why is this added at bottom? Check dates, they're more normally added at the top.

**Comment by ajakk** [comment]
Thanks all.

**Pull request closed without merging**